Atomic/syscontainers.py

```diff
@@ -306,6 +346,19 @@ def get_system_containers(self):
             ret.append(container)
         return ret
 
+    def delete_image(self, image):
+        repo = self._get_ostree_repo()
```

Comment:
I am fine with catching the IMAGE DOES NOT EXIST and returning success.  If the intent is to remove an object and the object does not exist, I consider that a success.